redesign/src/styles/layout/_404.scss

```diff
@@ -0,0 +1,129 @@
+.main.error-page {
+    margin-bottom: 160px;
+}
+
+.button {
+    &--option {
+        border: 1px solid $grey;
+        border-radius: 4px;
+        box-sizing: border-box;
+        cursor: pointer;
+        user-select: none;
+        @include padding (15px, 50px, 15px, 50px);
+        @include margin (15px, 30px, 25px, 30px);
+    }
+}
+
+a.button--option:hover {
```

Comment:
Kind of irregular and scattered use of the `&` throughout here. This could just as easily be included above by writing this classname as `&:hover` ?